src/Education/Commands/RuleDescriptionToolWindow.cs

```diff
@@ -36,18 +35,18 @@ internal class RuleDescriptionToolWindow : ToolWindowPane, IRuleDescriptionToolW
     {
         private const string ToolWindowIdAsString = "9E74B368-9FC3-47B0-A1C7-2DBA3A2C1762";
         public static readonly Guid ToolWindowId = new Guid(ToolWindowIdAsString);
-        private readonly FlowDocumentReader flowDocumentReader;
+        private readonly RuleDescriptionUserControl control;
 
         public RuleDescriptionToolWindow()
         {
             Caption = Resources.RuleDescriptionToolWindowCaption;
-            flowDocumentReader = new FlowDocumentReader();
-            Content = flowDocumentReader;
+            control = new RuleDescriptionUserControl();
+            Content = control;
```

Comment:
It now makes sense to display the `RuleDescriptionUserControl` rather than just creating a new reader.

The user control gives us a useful place to store the theming/styling info the document (see below). It also makes it easy to add boilerplate text for when the window is first opened.